assets/scripts/segments/SegmentForPalette.jsx

```diff
@@ -22,12 +23,14 @@ SegmentForPalette.propTypes = {
   connectDragSource: PropTypes.func,
   connectDragPreview: PropTypes.func,
   isIcon: PropTypes.bool,
+
   // Provided by parent
   type: PropTypes.string.isRequired,
   variantString: PropTypes.string.isRequired,
   onPointerOver: PropTypes.func,
   randSeed: PropTypes.number,
-  disabled: PropTypes.bool
+  disabled: PropTypes.bool,
+  tooltipTarget: PropTypes.object
 }
 
 function SegmentForPalette (props) {
```

Comment:
Awesome to see all the files deleted above and the code removed below! 

I don't know what SegmentForPalette means, though. My guess is the segment's representation in the palette (which is the thing at the bottom of the UI?)

There might be a better name for this, or at least a docstring for the dense. :-D

Reply:
Your guess is correct!